.github/workflows/dockerimage.yml

```diff
@@ -44,13 +44,13 @@ jobs:
         fi
         echo "::set-env name=HUB_TAG::${DOCKER_HUB_BASE_NAME}:${TAG_NAME}"
 
-    - name: Pull the Docker image
-      run: |
-        docker pull "${HUB_TAG}"
-
     - name: Build the Docker image
       run: |
-        docker build --cache-from "${HUB_TAG}" . --build-arg PYTHON_VERSION=${{ matrix.python_version }} --build-arg BUILD_TYPE=${{ matrix.build_type }} --file Dockerfile --tag "${HUB_TAG}"
+        if [ "${{ github.event_name }}" != 'release' ]; then
+          docker pull "${HUB_TAG}"
+          CACHE_FROM="--cache-from=${HUB_TAG}"
+        fi
+        docker build ${CACHE_FROM} . --build-arg PYTHON_VERSION=${{ matrix.python_version }} --build-arg BUILD_TYPE=${{ matrix.build_type }} --file Dockerfile --tag "${HUB_TAG}"
```

Comment:
Do we have to define `CACHE_FROM`, for instance as an empty string, in the case of releases?

Reply:
I think it is optional since the shell simply ignores undefined variables by default, but I added the missing `else` clause as you mentioned.

Reply:
You're right. Still, thanks for the change. I think it aids comprehensibility.